src/components/editor/GroupChoiceModal.jsx

```diff
@@ -32,6 +32,7 @@ const groupsToGroupValues = (groupIds, groupMap, ownerGroupId = null) =>
     }))
 
 const GroupChoiceModal = () => {
+  const errorKey = useAlerts()
```

Comment:
You'll see lots of changes like this -- get the error key from context instead of static import.